src/print.jl

```diff
@@ -272,13 +272,23 @@ end
 ## Constraints
 #------------------------------------------------------------------------
 
-function Base.show(io::IO, ref::ConstraintRef{Model})
+function Base.show(io::IO, ref::ConstraintRef)
```

Comment:
Please document the methods that a `ConstraintRef` needs to implement in order to avoid a `MethodError` on `show`.